pages/about/website-policies-and-notices.md

```diff
@@ -185,8 +185,6 @@ Federal agencies are required to provide the following links to agency-level inf
 
 -   [Office of the Inspector General](https://www.gsaig.gov/)
 
--   [Open government initiatives](https://www.gsa.gov/governmentwide-initiatives/gsas-open-government-initiatives)
```

Comment:
Possibly https://www.gsa.gov/governmentwide-initiatives/open-government/open-government-plan ?

Reply:
There's also https://www.gsa.gov/node/78692

Reply:
@mejiaj This is better. Thanks. [0a9a114](https://github.com/uswds/uswds-site/pull/1735/commits/0a9a114e1a6e83cf3a3c797206c97cd3000dc783)